okhttp/src/main/java/okhttp3/ConnectionSpec.java

```diff
@@ -50,10 +50,17 @@
       CipherSuite.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
   };
 
+  // TLS 1.3
+  private static final CipherSuite[] TLS13_CIPHER_SUITES = new CipherSuite[] {
```

Comment:
this is neat stuff.

should document that this also wants TLS_CHACHA20_POLY1305_SHA256 which is available in BoringSSL.